equinix/data_source_fabric_port_acc_test.go

```diff
@@ -0,0 +1,64 @@
+package equinix
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceFabricPort_PNFV(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ExternalProviders: testExternalProviders,
+		Providers:         testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceFabricPort("c4d85dbe-fa99-a999-f7e0-306a5c00af26"),
```

Comment:
At minimum I'd suggest moving this ID to a variable or a constant.  What we've done in similar situations in Metal acceptance tests is to read the ID from an environment variable so we can quickly change the ID if we're testing against a different account or if the resource we're depending on gets deleted accidentally.

Reply:
How do you handle the ENV variables? I didn't see any in the workflows for IDs. We would need to add a large amount in order to dynamically adjust the tests. I like the idea. What's the pattern you're doing for it right now?

Reply:
Actually we only have one resource we're tracking, a dedicated connection in our test account.  The ID is stored as a secret and passed in to the test job here: https://github.com/equinix/terraform-provider-equinix/blob/main/.github/workflows/acctest.yml#L110

The code that looks that up is spaghetti but these search results show the strands: https://github.com/search?q=repo%3Aequinix%2Fterraform-provider-equinix%20metalDedicatedConnIDEnvVar&type=code

Reply:
I see. 

Let's say we had 50+ id's or other variables that would be nice to adjust on the fly. That would get very hard to maintain in the workflow file especially when we have to add additional tests that have similar needs. 

Could we postpone coming up with a solution for this for another follow up PR?

Reply:
I like the ENV variable method for Ports. I think that's the only resource that actually needs to exist outside of what we can create in Terraform.

I'll do a quick follow up to this one with that included as I'll be adding additional acceptance tests very soon.

Reply:
One option might be to put a JSON string in the env var instead of a single ID, but I'm not sure what kind of length limitations you might run into for a single env var.  Since these IDs are being committed in the clear anyway, you don't need to use a secret for it.

Regardless of the approach you take, I'd still at least recommend defining the IDs in variables and referencing those, rather than hard-coding the same ID in multiple places.